views/service-offline-mid-journey.html

```diff
@@ -15,8 +15,11 @@ <h1 class="govuk-heading-l">
 
     <p class="govuk-body-m">{{ i18n.serviceErrorTryAgainLater }}</p>
     <p>{{ i18n.serviceErrorNotSavedAnswers }}</p>
-
-    <a href="https://www.gov.uk/contact-companies-house" data-event-id="500-error-page-contact-us">{{ i18n.serviceErrorContact }}</a> {{  i18n.serviceErrorIfAnyQuestions }}
+{% if lang === "en" %}
```

Comment:
maybe safer to special case cy and default to the en order of things? In unlikely case someone adds a new lang they would see no text otherwise. Maybe beter to have 3 i18n vars though: Before text, link text and after text. Then we don't need a if lang switch at all. The before would be empty for en and the after empty for welsh.